seshdash/views.py

```diff
@@ -240,3 +254,55 @@ class UserList(generics.ListAPIView):
 class UserDetail(generics.RetrieveAPIView):
     queryset = User.objects.all()
     serializer_class = UserSerializer
+
+# This function  get_high_chart_data is help to get object to use in the High Chart Daily PV production and cloud cover
```

Comment:
He Janvier not sure if you mis understood me. The comments should be inline in the code.

Everything else looks ok. Comment should explain the code that's it's nearby. You can also have a separate comment explaining the overall function as well.